mne/viz/_3d.py

```diff
@@ -1143,3 +1143,107 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
         fig.scene.x_plus_view()
 
     return fig
+
+
+def snapshot_montage(fig, montage, with_points=False):
```

Comment:
what would be more explicit? I could use "screenshot" instead of "snapshot" to match more with the `mlab` syntax. Or something like `take_3d_screenshot`?